scripts/download-binaries.sh

```diff
@@ -40,10 +40,11 @@ download_binaries() {
     if [[ $? -eq 0 ]]; then
         echo "Binary downloaded successfully"
 
-        tar -xzvf kwild.tar.gz './kwild'
-        mkdir -p ./.build
-        mv ./kwild .build
-        rm "./kwil-db.tar.gz"
+         mkdir -p ./kwild
+         tar -xzvf kwild.tar.gz -C ./kwild
+         mkdir -p ./.build
+         mv ./kwild .build
+         rm ./kwild.tar.gz
 
         chmod +x ./.build/kwild
```

Comment:
```suggestion
         tar -xzvf kwild.tar.gz -C ./.build
         rm ./kwild.tar.gz

        chmod +x ./.build
```

When using `task build`, the binaries are built from the `node` itself. This will place the file inside the `.build` directory.
It will be more direct this way and sync with `task build`. So there's no need to place it under `.build/kwild` directory.

Reply:
This would certainly work and would not require the change to Taskfile

Reply:
yes, correct.

Reply:
new commit added reflecting these changes